opentok-whiteboard.js

```diff
@@ -341,6 +386,12 @@ var OpenTokWhiteboard = angular.module('opentok-whiteboard', ['opentok'])
                         if (drawHistory.length > 0 && event.connection.connectionId !==
                                 OTSession.session.connection.connectionId) {
                             batchSignal('otWhiteboard_history', drawHistory, event.connection);
+                            // Iterate through redo stack to hide any paths which have been undone
+                            if (redoStack.length) {
+                                redoStack.forEach(function(id) {
+                                    sendUpdate('otWhiteboard_undo', id);
```

Comment:
Ok, added an arg to send the update just to that connection.

The undo operations use paper.js's path id's. They have been working for me when opening up a couple connections. Because the paths are created from history in the same order they were drawn, and paper.js id's are sequential they match up across clients.